### PR TITLE
[Rule Tuning] User Added to Privileged Group

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

Related to #3005 

## Summary

Expands the rule to detect modifications to these groups on non-English AD envs.

### CODE REVIEW

File: rules/windows/persistence_user_account_added_to_privileged_group_ad.toml [hunk 1]
```diff
@@ -73,14 +73,32 @@ type = "eql"
 
 query = '''
 iam where winlog.api:"wineventlog" and event.action == "added-member-to-group" and
```
Comment (terrancedejesus) [inline rules/windows/persistence_user_account_added_to_privileged_group_ad.toml:75]:
```suggestion
iam where winlog.api == "wineventlog" and event.action == "added-member-to-group" and
```

File: rules/windows/persistence_user_account_added_to_privileged_group_ad.toml [hunk 2]
```diff
@@ -73,14 +73,32 @@ type = "eql"
...
-                "Local Administrators",
-                "Domain Admins",
-                "Enterprise Admins",
-                "Backup Admins",
-                "Schema Admins",
-                "DnsAdmins",
-                "Exchange Organization Administrators")
+(
+    (
+        group.name : (
```
Comment (terrancedejesus) [inline rules/windows/persistence_user_account_added_to_privileged_group_ad.toml:78]: I see that this is mainly admin focused, but here are some additional considerations maybe?

```
"Account Operators",
"Server Operators",
"Print Operators",
"Remote Desktop Users",
"Power Users",
"Hyper-V Administrators",
"Remote Management Users",
"Cryptographic Operators",
"Network Configuration Operators",
```

```
"S-1-5-32-548",
"S-1-5-32-549",
"S-1-5-32-550",
"S-1-5-32-555",
"S-1-5-32-547",
"S-1-5-32-556",
"S-1-5-32-569",
"S-1-5-32-580",
```

**Review by terrancedejesus — APPROVED** [review]
Left a couple ideas and suggestions. Up to you, thanks for tuning!

File: rules/windows/persistence_user_account_added_to_privileged_group_ad.toml (hunk 2 above)
Comment (w0rk3r) [reply rules/windows/persistence_user_account_added_to_privileged_group_ad.toml:78]: Just added a few more that were missing

File: rules/windows/persistence_user_account_added_to_privileged_group_ad.toml [hunk 3]
```diff
@@ -73,14 +73,32 @@ type = "eql"
...
+        group.id : (
+            "S-1-5-32-544",
+            "S-1-5-32-544",
+            "S-1-5-21-*-512",
+            "S-1-5-21-*-519",
+            "S-1-5-21-*-551",
+            "S-1-5-21-*-518",
+            "S-1-5-21-*-1101",
+            "S-1-5-21-*-1102"
+        )
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/persistence_user_account_added_to_privileged_group_ad.toml]: > 1101 and 1102 

Not sure either, but these are commonly related to the DNS Admins group, but it seems that there is no "official" RID for it

[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/persistence_user_account_added_to_privileged_group_ad.toml]: >better use endswith for performance when using multi wildcards checks

Isn't that the same as using the wildcard? I think this was the conclusion

**Pull request merged**